planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/include/obstacle_avoidance_planner/node.h

```diff
@@ -103,113 +87,107 @@ class ObstacleAvoidancePlanner
-  std::unique_ptr<autoware_perception_msgs::DynamicObjectArray> in_objects_ptr_;
-
-  dynamic_reconfigure::Server<obstacle_avoidance_planner::AvoidancePlannerConfig>
-    dynamic_reconfigure_srv_;
+  std::unique_ptr<std::vector<autoware_planning_msgs::msg::PathPoint>> prev_path_points_ptr_;
+  std::unique_ptr<autoware_perception_msgs::msg::DynamicObjectArray> in_objects_ptr_;
 
   // TF
   std::unique_ptr<tf2_ros::Buffer> tf_buffer_ptr_;
   std::unique_ptr<tf2_ros::TransformListener> tf_listener_ptr_;
-  std::unique_ptr<ros::Time> prev_replanned_time_ptr_;
+  std::unique_ptr<rclcpp::Time> prev_replanned_time_ptr_;
 
   // ROS
-  ros::NodeHandle nh_, pnh_;
-  ros::Publisher trajectory_pub_;
-  ros::Publisher avoiding_traj_pub_;
-  ros::Publisher debug_smoothed_points_pub_;
-  ros::Publisher is_avoidance_possible_pub_;
-  ros::Publisher debug_markers_pub_;
-  ros::Publisher debug_clearance_map_pub_;
-  ros::Publisher debug_object_clearance_map_pub_;
-  ros::Publisher debug_area_with_objects_pub_;
-  ros::Subscriber path_sub_;
-  ros::Subscriber twist_sub_;
-  ros::Subscriber objects_sub_;
-  ros::Subscriber is_avoidance_sub_;
+  rclcpp::Publisher<autoware_planning_msgs::msg::Trajectory>::SharedPtr trajectory_pub_;
+  rclcpp::Publisher<autoware_planning_msgs::msg::Trajectory>::SharedPtr avoiding_traj_pub_;
+  rclcpp::Publisher<autoware_planning_msgs::msg::Trajectory>::SharedPtr debug_smoothed_points_pub_;
+  rclcpp::Publisher<std_msgs::msg::Bool>::SharedPtr is_avoidance_possible_pub_;
+  rclcpp::Publisher<visualization_msgs::msg::MarkerArray>::SharedPtr debug_markers_pub_;
+  rclcpp::Publisher<nav_msgs::msg::OccupancyGrid>::SharedPtr debug_clearance_map_pub_;
+  rclcpp::Publisher<nav_msgs::msg::OccupancyGrid>::SharedPtr debug_object_clearance_map_pub_;
+  rclcpp::Publisher<nav_msgs::msg::OccupancyGrid>::SharedPtr debug_area_with_objects_pub_;
+  rclcpp::Subscription<autoware_planning_msgs::msg::Path>::SharedPtr path_sub_;
+  rclcpp::Subscription<geometry_msgs::msg::TwistStamped>::SharedPtr twist_sub_;
+  rclcpp::Subscription<autoware_perception_msgs::msg::DynamicObjectArray>::SharedPtr objects_sub_;
+  rclcpp::Subscription<std_msgs::msg::Bool>::SharedPtr is_avoidance_sub_;
 
   // callback functions
-  void pathCallback(const autoware_planning_msgs::Path & msg);
-  void twistCallback(const geometry_msgs::TwistStamped & msg);
-  void objectsCallback(const autoware_perception_msgs ::DynamicObjectArray & msg);
-  void enableAvoidanceCallback(const std_msgs::Bool & msg);
-  void configCallback(
-    const obstacle_avoidance_planner::AvoidancePlannerConfig & config, const uint32_t level);
+  void pathCallback(const autoware_planning_msgs::msg::Path::SharedPtr);
+  void twistCallback(const geometry_msgs::msg::TwistStamped::SharedPtr);
+  void objectsCallback(const autoware_perception_msgs::msg::DynamicObjectArray::SharedPtr);
+  void enableAvoidanceCallback(const std_msgs::msg::Bool::SharedPtr);
```

Comment:
replace to new message.

Reply:
done in https://github.com/tier4/Pilot.Auto/pull/83/commits/4e76eec31e00ad4c8bb426ae6d9eefa05e758dad and https://github.com/tier4/Pilot.Auto/pull/83/commits/0f2106a0e145a90208c69326dc830b081874de6d